cpuid/src/transformer/amd.rs

```diff
@@ -60,8 +60,6 @@ mod test {
 
     #[test]
     fn test_update_extended_cache_topology_entry() {
-        use cpu_leaf::leaf_0x8000001d::*;
```

Comment:
FYI: I was already doing this in #1039 , but maybe yours gets merged first.

Reply:
Yeah, one of them will just automagically disappear in a merge.